src/types.ts

```diff
@@ -42,11 +42,11 @@ const LaxPresetNames  = [
   "flipX" ,
   "flipY" ,
   "blurIn" ,
-  "blurOur" ,
+  "blurOut" ,
   "blurInOut"
 ]
 export const isPresetName = (presetName: string): presetName is LaxPresetName => LaxPresetNames.indexOf(presetName) !== -1 ? true : false;
-
+export type LaxPresetFn = (x: number | string, y: number | string) => LaxPresetStyleProps
```

Comment:
The presetFunction receive the variables "y" and "str"  
```
  fadeInOut: (y = 30, str = 0) => ...
```

Reply:
From a type perspective, this would not be a problem. Because the `LaxPresetFn` expects two arguments with each either being a number or a string, the fadeInOut preset function passes the type check.
I can change the type definitition to be: `export type LaxPresetFn = (arg1: number | string, arg2: number | string) => LaxPresetStyleProps` to increase readability?